content/docs/studio/user-guide/prepare-your-repositories.md

```diff
@@ -53,5 +53,19 @@ Datasets, metrics, and hyperparameters can be added to a project in two ways:
    file (within your Git repo) which contains your saved metrics. Refer to the
    section [on project settings] to learn how to specify the custom files.
 
+## Prepare Your Repositories to Run New Experiments
+
+To prepare your repositories to run new experiments from Iterative Studio, you
+should integrate them with a CI/CD setup that includes model training process.
```

Comment:
```suggestion
should integrate them with a CI/CD setup that includes an arbitrary job execution process.
```

Reply:
I'm not changing this, coz to successfully run experiments, the users do need to include a model training process in their CI workflow. Yes, we can't enforce that they will do model training, but if they don't, then it won't really run experiments. So even though the job could be an arbitrary job, our suggestion to users should be to include model training. @mvshmakov Pls let me know if you disagree.